app/view_models/forgot_password_show.rb

```diff
@@ -0,0 +1,26 @@
+class ForgotPasswordShow
+  def initialize(params:, session:)
```

Comment:
I find it just as easy / easier to access those within the model itself. What do you see as the benefits or passing the parsed vals instead?